pkgs/development/python-modules/dbus-python/default.nix

```
@@ -90,7 +90,7 @@ lib.fix (
     checkPhase = ''
       runHook preCheck
 
-      meson test -C _meson-build --no-rebuild --print-errorlogs
+      meson test -C _meson-build --no-rebuild --print-errorlogs --timeout-multiplier 0
```

```suggestion
      meson test -C _meson-build --no-rebuild --print-errorlogs
```

not required due to https://github.com/NixOS/nixpkgs/pull/356589

---

It is required. That PR only modifies the default mesonFlags, this is a custom meson invocation.